Spellcheck pt. 2 (#6437)
braze-inc/braze-docs: _docs/_api/objects_filters/recipient_object.md

## Recipient object deduping

> When making an API call with the Recipient Object, **if there exists a duplicated recipient targeting the same address (i.e., email, push), the user will be deduped**, meaning identical users will be removed, leaving one. 

Author: ```suggestion
When making an API call with the Recipient Object, **if there exists a duplicated recipient targeting the same address (that is, email, push), the user will be deduped**, meaning identical users will be removed, leaving one. 
```
